lib/helpers.js

```diff
@@ -850,22 +850,9 @@ function toAvdLocaleArgs (language, country) {
 async function getAndroidPrefsRoot () {
   // https://gerrit.pixelexperience.org/plugins/gitiles/development/+/5c11852110eeb03dc5a69111354b383f98d15336/tools/androidprefs/src/com/android/prefs/AndroidLocation.java
   let location = null;
-  if (system.isWindows()) {
-    let localAppData = process.env.LOCALAPPDATA;
-    if (!localAppData) {
-      localAppData = process.env.USERPROFILE;
-      if (localAppData) {
-        localAppData = path.resolve(localAppData, 'Local Settings', 'Application Data');
-      }
-    }
-    if (localAppData) {
-      location = path.resolve(localAppData, 'Android');
-    }
-  } else {
-    const home = process.env.HOME;
-    if (home) {
-      location = path.resolve(home, '.android');
-    }
+  const home = process.env.HOME || process.env.USERPROFILE;
```

Comment:
I think it would then make sense to read and check ANDROID_AVD_HOME and ANDROID_EMULATOR_HOME variables first. Also, it is not very clear what is HOME in Windows. Does it always equal to USERPROFILE value?

Reply:
Good point, I've added checks for `$ANDROID_AVD_HOME`, `$ANDROID_EMULATOR_HOME` and `$ANDROID_SDK_HOME`